xdp-dump/xdpdump.c

```diff
@@ -55,6 +56,7 @@ static const struct dumpopt {
 	struct iface          iface;
 	uint32_t              snaplen;
 	char                 *pcap_file;
+	char                 *multiprogs;
```

Comment:
Why is this still called 'multiprogs'? :)

Reply:
good question changed it :)